.travis.yml

```diff
@@ -29,10 +24,18 @@ matrix:
     fast_finish: true
     include:
         - php: 5.5
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_VERSION=2.8.*
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+        - php: 7.1
+          env: DEPENDENCIES=dev
+          # Test against LTS versions
+        - php: 7.0
+          env: SYMFONY_VERSION=2.8.*
+
+before_install:
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
```

Comment:
Maybe we should change this to require only useful bundles instead of the whole symfony?